build.gradle.kts

```diff
@@ -0,0 +1,214 @@
+import net.researchgate.release.GitAdapter.GitConfig
+import java.util.Date
+import org.gradle.api.tasks.testing.logging.TestExceptionFormat.FULL
+
+/*
+ * Copyright 2017 Google LLC. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+plugins {
+  id("java")
+  id("maven")
+  id("java-gradle-plugin")
+  id("net.researchgate.release") version "2.6.0"
+  id("com.github.sherter.google-java-format") version "0.8"
+  id("checkstyle")
+  id("jacoco")
+}
+
+repositories {
+  mavenCentral()
+}
+
+java {
+  sourceCompatibility = JavaVersion.VERSION_1_8
+  targetCompatibility = JavaVersion.VERSION_1_8
+}
+
+group = "com.google.cloud.tools"
+
+dependencies {
+  compile(localGroovy())
+  compile(gradleApi())
+  compile("com.google.cloud.tools:appengine-plugins-core:0.9.8")
+
+  testCompile("commons-io:commons-io:2.4")
+  testCompile("junit:junit:4.12")
+  testCompile("org.hamcrest:hamcrest-library:1.3")
+  testCompile("org.mockito:mockito-core:2.23.4")
+}
+
+
+tasks.wrapper {
+  gradleVersion = "6.9"
```

Comment:
Why the older version?

Reply:
6.9 is the last Gradle version that still has the (now removed) "maven" plugin. 
This PR is basically taken from https://github.com/GoogleCloudPlatform/app-gradle-plugin/pull/398 and doing less things at a time but I'm happy to switch to https://github.com/GoogleCloudPlatform/app-gradle-plugin/pull/398 directly